internal/servers/controller/testing.go

```diff
@@ -414,6 +416,17 @@ func NewTestController(t *testing.T, opts *TestControllerOpts) *TestController {
 		t.Fatal(err)
 	}
 
+	tc.b.DevLoopbackHostPluginId = DefaultTestPluginId
+	plg := plugin.NewWrappingPluginClient(plugin.NewLoopbackPlugin())
+	plugOpts := []hostplugin.Option{
+		hostplugin.WithDescription("Provides an initial loopback host plugin in Boundary"),
+		hostplugin.WithPublicId(tc.b.DevLoopbackHostPluginId),
+	}
+	if _, err = tc.b.RegisterHostPlugin(ctx, "loopback", plg, plugOpts...); err != nil {
```

Comment:
Please remove the part of this file which adds the loopback plugin already (see the section close to `CreateDevDatabase` in TestControllerConfig)

Reply:
TIL we already have loopback in test controllers!